Library/Formula/texpretty.rb

```diff
@@ -0,0 +1,23 @@
+class Texpretty < Formula
+  desc "texpretty - prettyprint TeX files"
```

Comment:
The `desc` output will include the formula name already, so this would be fine with `desc "Prettyprint TeX files"` :)